lib/interface/cli/helpers/api.js

```diff
@@ -0,0 +1,24 @@
+const request = require('requestretry');
+const _ = require('lodash');
+
+async function getApiVersion(token) {
+    const RequestOptions = {
+        url: 'https://g.codefresh.io/api/user',
+        headers: {
+            Authorization: token,
+        },
+        json: true,
+        // Add retry options to the RequestOptions object
+        maxAttempts: 3, // Retry up to 3 times
+    };
+    try {
+        const res = await request(RequestOptions);
+        return _.get(res, 'headers.api_version');
+    } catch (error) {
```

Comment:
it might be useful to add here some debug logging instead of just swallowing the error. we should use `debug` lib like its done in other places in the cli, but keep in mind that `debug` is used with different namespaces for different areas in the code and we want to keep them unique.